encord/orm/group.py

```diff
@@ -0,0 +1,44 @@
+from datetime import datetime
+from uuid import UUID
+
+from encord.orm.base_dto import BaseDTO
+from encord.orm.dataset import DatasetUserRole
+from encord.utilities.ontology_user import OntologyUserRole
+from encord.utilities.project_user import ProjectUserRole
+
+
+class Group(BaseDTO):
+    group_hash: str
```

Comment:
In original API, this is a UUID, not a string. But I think here we want it to be a str, to make it easier for users. They don't have to convert their strings into UUID's.

Reply:
Well, if we follow 'be strict with what you outputs but relaxed with your inputs', I would say this should be UUID but the 'add_group' and 'remove_group' should take `Union[UUID, str]` and convert internally.

Maybe ask Sergei's opinion here? He's the semi-official king of the SDK.